### PR TITLE
Allow to compile Kopia cli on OpenBSD

### PR DESCRIPTION
**Pull request by ledeuns**
Only CLI is available and mount is not supported.

### CODE REVIEW

File: fs/localfs/local_fs_openbsd.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package localfs
```
Comment (jkowalski) [inline fs/localfs/local_fs_openbsd.go:1]: doesn't `local_fs_nonwindows.go` work here?

Comment (jkowalski) [reply fs/localfs/local_fs_openbsd.go:1]: you probably just need to tweak build flags:

`local_fs_64bit.go` by adding  !openbsd`
`local_fs_32bit.go` by adding `openbsd`

**Comment by jkowalski** [comment]
also, can you add `.goreleaser.yml` entry similar to freebsd

File: fs/localfs/local_fs_openbsd.go (hunk 1 above)
Comment (ledeuns) [reply fs/localfs/local_fs_openbsd.go:1]: Doing that yields : 
```
# github.com/kopia/kopia/fs/localfs
fs/localfs/local_fs_nonwindows.go:26:12: undefined: platformSpecificWidenDev
fs/localfs/local_fs_nonwindows.go:27:13: undefined: platformSpecificWidenDev
```

Comment (ledeuns) [reply fs/localfs/local_fs_openbsd.go]: Thank you, it is fixed.

**Review by julio-lopez — COMMENTED** [review]
👍 LGTM, but @jkowalski should approve.

File: .goreleaser.yml [hunk 2]
```diff
@@ -26,6 +26,7 @@ archives:
       linux: linux
       windows: windows
       freebsd: freebsd-experimental
+      openbsd: openbsd-experimental
```
Comment (jkowalski) [inline .goreleaser.yml:29]: you also want this in `goos` section above. test with `make goreleaser`

Comment (ledeuns) [reply .goreleaser.yml:30]: `make goreleaser` fails due to `goreleaser` not having an OpenBSD release package :)

**Pull request merged**